frontend/src/components/Home/Home.js

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import '../../styles/Home/Home.scss';
 import Form from 'react-bootstrap/Form';
 import Tab from 'react-bootstrap/Tab';
 import Tabs from 'react-bootstrap/Tabs';
-import { Redirect, useHistory } from 'react-router-dom';
-import Icon from '../../icons/user-profile.png';
-import graphIcon from '../../icons/graph-icon.png';
+import Modal from 'react-bootstrap/Modal';
+import Carousel from 'react-bootstrap/Carousel';
+import { Redirect } from 'react-router-dom';
+import Icon from '../../assets/icons/user-profile.png';
+import graphIcon from '../../assets/icons/graph-icon.png';
 
 function Home() {
-    const history = useHistory();
-    console.log(history);
+    const [tutorial, showTutorial] = useState(true);
```

Comment:
Does this show every time you go to the home page

Reply:
Actually honestly it should be fine to just have this; I'd rather not update the user model to add a field for this